frontend/packages/db-structure/src/schema/dbStructure.ts

```diff
@@ -43,15 +43,31 @@ const tableSchema = v.object({
 
 export type Table = v.InferOutput<typeof tableSchema>
 
+const cardinalitySchema = v.union([
+  v.literal('ONE_TO_MANY'),
+  v.literal('ONE_TO_ONE'),
+])
+const foreignKeyConstraintSchema = v.union([
+  v.literal('CASCADE'),
+  v.literal('RESTRICT'),
+  v.literal('SET NULL'),
+  v.literal('SET DEFAULT'),
+  v.literal('NO ACTION'),
```

Comment:
I thought it would be good to unify with UPPER_CAMEL_CASE. What do you think?

```suggestion
  v.literal('SET_NULL'),
  v.literal('SET_DEFAULT'),
  v.literal('NO_ACTION'),
```

Reply:
Yeah, I was torn between `SET NULL` or `SET_NULL`.
I decided `SET NULL` because values can be used in frontend view as is.
I thought it would be better to display it with spaces for user.

Is it necessary to use it for the frontend view as is?

Reply:
The format as data and the label as UI should be considered separately.
This is because there are many possible ways of expression, such as trial and error in UX writing or the use of icons.

Reply:
Fixed 👍:
9bbf00668b87d236ab12764d26c3e6c0e8671c10

Reply:
thanks 👍🏻